src/main/io/gps.c

```diff
@@ -247,11 +280,39 @@ typedef struct ubxCfgNav5_s {
     uint8_t reserved1[5];
 } ubxCfgNav5_t;
 
+typedef struct ubxCfgNav5x_s {
+    uint16_t version;
+    uint16_t mask1;
+    uint32_t mask2;
+    uint8_t reserved0[2];
+    uint8_t minSVs;
+    uint8_t maxSVs;
+    uint8_t minCNO;
+    uint8_t reserved1;
+    uint8_t iniFix3D;
+    uint8_t reserved2[2];
+    uint8_t ackAiding;
+    uint16_t wknRollover;
+    uint8_t sigAttenCompMode;
+    uint8_t reserved3;
+    uint8_t reserved4[2];
+    uint8_t reserved5[2];
+    uint8_t usePPP;
+    uint8_t aopCfg;
+    uint8_t reserved6[2];
+    uint8_t reserved7[4];
+    uint8_t reserved8[3];
+    uint8_t useAdr;
+} ubxCfgNav5x_t;
+
 typedef union ubxPayload_s {
     ubxPollMsg_t poll_msg;
     ubxCfgMsg_t cfg_msg;
     ubxCfgRate_t cfg_rate;
+    ubxCfgValSet_t cfg_valset;
+    ubxCfgValGet_t cfg_valget;
```

Comment:
```suggestion
    ubxCfgValGetSet_t cfg_valGetSet;
```

Reply:
ubxPayload_s is already union

Reply:
I didn't see that. Reviewing in the browser instead of in vscode is a bit confusing sometimes, my bad.